addon/templates/components/polaris-description-list.hbs

```diff
@@ -1,16 +1,18 @@
-{{#each items as |item|}}
-  <dt class="Polaris-DescriptionList__Term">
-    {{#if item.termComponent}}
-      {{component item.termComponent}}
-    {{else}}
-      {{render-content item.term}}
-    {{/if}}
-  </dt>
-  <dd class="Polaris-DescriptionList__Description">
-    {{#if item.descriptionComponent}}
-      {{component item.descriptionComponent}}
-    {{else}}
-      {{render-content item.description}}
-    {{/if}}
-  </dd>
-{{/each}}
+<dl class="Polaris-DescriptionList">
+  {{#each @items as |item|}}
+    <dt class="Polaris-DescriptionList__Term">
+      {{#if item.termComponent}}
+        {{component item.termComponent}}
```

Comment:
Since we've removed the deprecation warning for `termComponent`, do we want to drop support here? Same below re. `descriptionComponent` too.

Reply:
oh, for sure
I thought it's just some leftovers